controller/envs.go

```diff
@@ -29,10 +29,13 @@ func (c *Controller) GetEnvs(ctx context.Context) (*entity.Envs, error) {
 
 	if val, ok := projectCfg.LockedEnvsNames[projectCfg.Environment]; ok && val {
 		fmt.Println(ui.Bold(ui.RedText("Protected Environment Detected!").String()))
-		err := ui.PromptConfirm("Press Enter to Confirm Action")
+		confirm, err := ui.PromptYesNo("Continue fetching variables?")
 		if err != nil {
 			return nil, err
 		}
+		if !confirm {
+			return nil, err
```

Comment:
If not confirm should we return `nil, nil`?
```suggestion
			return nil, nil
```